spring-integration-core/src/main/java/org/springframework/integration/config/annotation/AbstractMethodAnnotationPostProcessor.java

```diff
@@ -243,6 +243,10 @@ protected String getInputChannelAttribute() {
 		return INPUT_CHANNEL_ATTRIBUTE;
 	}
 
+	protected ConfigurableListableBeanFactory getBeanFactory() {
+		return this.beanFactory;
```

Comment:
Yeah - this is some left-over dross from when I was trying to use the BF too early.